internal/module/kernelmapper.go

```diff
@@ -19,14 +19,11 @@ const (
 )
 
 type NodeOSConfig struct {
-	kernelFullVersion  string `subst:"KERNEL_FULL_VERSION"`
-	kernelVersionMMP   string `subst:"KERNEL_XYZ"`
-	kernelVersionMajor string `subst:"KERNEL_X"`
-	kernelVersionMinor string `subst:"KERNEL_Y"`
-	kernelVersionPatch string `subst:"KERNEL_Z"`
-	kernelRHELVersion  string `subst:"KERNEL_RHEL_VERSION"`
```

Comment:
Don't we need RHEL's  info here?

Reply:
i am currently just backporting. Once i have the basics in, i will need to add changes to the downstream, and that will include all the needed changes

Reply:
Ack.